src/python-bindings/schedd.cpp

```diff
@@ -4713,6 +4731,24 @@ void export_schedd()
             :param str args: The arguments to pass to the ``QUEUE`` statement.
             )C0ND0R",
             boost::python::args("self", "args"))
+        .def("setSubmitMethod", &Submit::setSubmitMethod,
+            R"C0ND0R(
+            Sets the Job Ad attribute ``JobSubmitMethod`` to passed over number. Set
+            number must be greater than or equal to ``100``. If no number is passed then
+            ``JobSubmitMethod`` will be undefined. **Note~** Setting of ``JobSubmitMethod``
+            must occur before job is submitted to Schedd.
+
+            :param int value: Value of ``JobSubmitMethod``.
+            )C0ND0R",
+            (boost::python::arg("self"), boost::python::arg("value")=-1))
+        .def("_setSubmitMethod", &Submit::_setSubmitMethod,
+            (boost::python::arg("self"), boost::python::arg("value")=-1, boost::python::arg("internal")=false))
+        .def("getSubmitMethod", &Submit::getSubmitMethod,
```

Comment:
I want Jason to look at this and approve the strategy of having an unducumented internal method for CLI